tests/utils/postprocess/test_math.py

```diff
@@ -85,33 +85,53 @@ def test_token():
     assert len(t) == 0
     assert repr(t) == "''"
     assert t == ''
+    assert t.get_text() == ''
     t1 = Token(' a ', quoted=True)
+    assert t1.get_text() == '`a`'
     t2 = Token('a')
     assert t1 == t2
 
 
+# DEPRECATED: OLD SYNTAX
 def test_parse_formula():
-    assert _parse_formula('a') == ['a']
-    assert _parse_formula('a+b') == ['a', '+', 'b']
-    assert _parse_formula('pika + chuuu') == ['pika', '+', 'chuuu']
-    assert _parse_formula('pika + (chuuu/10)') == ['pika', '+', '(', 'chuuu', '/', '10', ')']
-    assert _parse_formula('pika + (chu uu/10)') == ['pika', '+', '(', 'chu uu', '/', '10', ')']
-    assert _parse_formula('pika + (chu_uu/10)') == ['pika', '+', '(', 'chu_uu', '/', '10', ')']
-    assert _parse_formula('pika + ("chu-uu"/10)') == ['pika', '+', '(', 'chu-uu', '/', '10', ')']
-    assert _parse_formula('a + b*3.1') == ['a', '+', 'b', '*', '3', '.', '1']
-    assert _parse_formula('a + "b*3.1"') == ['a', '+', 'b*3.1']
-    assert _parse_formula('("and-another" - yet_another) / (and - another)') == [
+    assert _parse_formula('a', quote_chars=("'", '"')) == ['a']
```

Comment:
What about adding 2 explicit constants `DEPRECATED_COLUMN_QUOTE_CHARS` and `COLUMN_QUOTE_CHARS`?